splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/BaseRoutes.java

```diff
@@ -0,0 +1,98 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.eclipse.microprofile.config.Config;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
+public class BaseRoutes extends ErrorHandlingBaseRoutes {
```

Comment:
This should extend from `EndpointRouteBuilder` as the `ErrorHandlingBaseRoutes` are not defined yet.